docs/http/traffic-policy/actions/remove-headers.mdx

```diff
@@ -0,0 +1,33 @@
+import { ExampleHTTP } from "/examples/actions/remove-headers.mdx";
+
+# Remove Headers
+
+## Overview
+
+The Remove Headers policy action enables the removal of headers to both incoming requests and outbound responses.
+
+## Example
+
+<ExampleHTTP />
+
+## Behavior
+
+When executed as an inbound policy, this action will remove headers on an incoming http request before reaching the upstream server with
+the configured headers. When executed as an outbound policy, the configured headers are removed to the response from the upstream server.
+
+## Reference
+
+### Supported Directions
+
+- Inbound
+- Outbound
+
+### Configuration
+
+| Type             |
+| ---------------- |
+| `remove-headers` |
+
+| Parameter | &nbsp;              | Description                                               |
+| --------- | ------------------- | --------------------------------------------------------- |
+| `headers` | Array&lt;string&gt; | Headers to be removed to be added to request or response. |
```

Comment:
I think the description text should be `Headers to be removed from the request or response.`